codecov.yml

```diff
@@ -1 +1,6 @@
 comment: false
+
+codecov:
+  notify:
+    after_n_builds: 8
```

Comment:
The `after_n_builds` setting was set to 8 here because that's how many `coverage` jobs there are in CI?  That sounds brittle — it would be good to include a short comment about this.

Reply:
Yep, no way for codecov to detect this by itself, unfortunately. Added a comment.